ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleBannerAd.java

```diff
@@ -33,11 +32,6 @@
    */
```

Comment:
We should also update this comment similar to: `Vungle ad object for banner ads.`

Reply:
Resolved.